### ISSUE TITLE
New repository - This is no longer available

### ISSUE THREAD

**Issue by michelpereira** [leereilly/games#629]
Hi, there.

Because of no responses from the repository owner, [sindresorhus](https://github.com/sindresorhus/awesome/issues/1198#issuecomment-1000564663) gave me the permission to fork this repository and be the new maintainer.

Please move all your PR and Issues to:
https://github.com/michelpereira/awesome-games/

Thanks

**Comment by testman42** [comment]
wouldn't it then be good if the README.md in this repo had the link to the new repo at the top?
and additionally make this repo "archived"?

**Comment by JonnyTech** [comment]
Only @leereilly can do that and they seem AWOL.

**Comment by michelpereira** [comment]
@testman42 , yes. That would be good but the owner of the repository is no longer answering.

**Comment by testman42** [comment]
> Only @leereilly can do that and they seem AWOL.

Seems to post on Twitter every now and then - https://twitter.com/leereilly

Maybe someone who has Twitter account can try to contact him there. Or maybe we can try to get a hold of him on some other platform?
